xcode/libmoai/libmoai.xcodeproj/project.pbxproj

```diff
@@ -10652,7 +10651,6 @@
 				);
 				OTHER_CFLAGS = "";
 				PRODUCT_NAME = "moai-osx-zlcore";
-				VALID_ARCHS = "i386 x86_64";
```

Comment:
This was redundant, since the base xcconfig already had this setting with this same value.